CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## Unreleased
 
 ### Added
+- [#341](https://github.com/equinor/flownet/pull/341) Added option to add a min permeability threshold for which tubes are deactivated when the permeability is below this value.
```

Comment:
Add a comment here on how this new threshold value relates to / should be used with the minimum permeability value defined in the config file. See my comment in the PR page

Reply:
I think this is up to the user. You wouldn't set it equal to the minimum from the distribution otherwise you'll never remove anything....